nuget.config

```diff
@@ -5,11 +5,12 @@
   </config>
   <packageSources>
     <clear />
-    <add key="GVFSBuildDependencies" value="https://www.myget.org/F/gvfsbuilddependencies/api/v3/index.json" />
+    <add key="VFSForGitBuildDependencies" value="https://vfsforgit.myget.org/F/build-dependencies/api/v3/index.json" />
+    <add key="ArchivedGVFSBuildDependencies" value="https://vfsforgit.myget.org/F/gvfsbuilddependencies/api/v3/index.json" />

```

Comment:
Why do we need this key? The new feed has all of our latest packages, right? And the older commits still reference the original feed and it's not going away?

Reply:
`vfsforgit.myget.org/F/build-dependencies` is currently empty.

`vfsforgit.myget.org/F/gvfsbuilddependencies` is what was migrated from `www.myget.org/F/gvfsbuilddependencies`.

I guess another option would be to rename `gvfsbuilddependencies` but I couldn't figure out how to do that in a couple minutes of poking around the myget ui. Given that, I wanted to take this opportunity to move away from the gvfs name.

Reply:
I think we should just migrate the latest version of each package over to the new feed, and drop this archived feed from our code altogether. There's not much value in preserving two feeds here.

Reply:
Makes sense. I'll make those changes.

Reply:
I've moved the packages that I think are needed to build this commit. CI should be able to fact check me :)